packages/imagemagick7.rb

```diff
@@ -51,6 +45,7 @@ def self.patch
   end
 
   def self.build
+    FileUtils.ln_s "#{CREW_PREFIX}/include/locale.h", "#{CREW_PREFIX}/include/xlocale.h" 
     system "CFLAGS=' -I#{CREW_PREFIX}/include/gdk-pixbuf-2.0 -I#{CREW_PREFIX}/include/c++/v1/support/xlocale' \
            ./configure \
            --prefix=#{CREW_PREFIX} \
```

Comment:
You could replace `--prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}` with `#{CREW_OPTIONS}`.

Reply:
I'll do this. Didn't see that ... oops